openapi/wordpress/openapi.yml

```diff
@@ -0,0 +1,3008 @@
+openapi: 3.0.1
+info:
+  title: Wordpress v2 API
+  description: >
+    This is a generated connector for [WordPress API v1.0](https://developer.wordpress.org/rest-api/) OpenAPI specification.
+    
+    The WordPress REST API provides an interface for applications to interact with your WordPress site. 
+    WordPress API provides a rich set of tools and interfaces for building sites, and you should not feel pressured to use the REST API 
+    if your site is already working the way you expect. You do not need to use the REST API to build a WordPress theme or plugin.
+  x-ballerina-init-description: 
+    The connector initialization requires setting the API credentials.
+
+    Create a [WordPress account](https://login.wordpress.org/register) and obtain tokens by following [this guide](https://developer.wordpress.org/rest-api/using-the-rest-api/authentication/).
+  x-ballerina-display:
+    label: WordPress
+    iconPath: "resources/wordpress.svg"
+  version: 0.1.0
+paths:
+  /posts:
+    get:
+      summary: List Posts
+      operationId: getPosts
+      description: Scope under which the request is made; determines fields present
+        in response.
+      externalDocs:
+        description: More information here
+        url: http://v2.wp-api.org/reference/posts/
+      parameters:
+      - name: context
+        in: query
+        description: Scope under which the request is made; determines fields present
+          in response.
+        schema:
+          type: string
+          enum:
+          - view
+          - embed
+          - edit
+      requestBody:
```

Comment:
According to OpenAPI specification v3 [GET, DELETE and HEAD are no longer allowed to have request body because it does not have defined semantics as per RFC 7231](https://swagger.io/docs/specification/describing-request-body/#:~:text=GET%2C%20DELETE%20and%20HEAD%20are%20no%20longer%20allowed%20to%20have%20request%20body%20because%20it%20does%20not%20have%20defined%20semantics%20as%20per%20RFC%207231.). Not validating the GET request with request body is an issue found in the swagger parser we are using in Ballerina OpenAPI tool. Please check the validity of the usage.

Reply:
Added an issue at `ballerina-openapi` https://github.com/ballerina-platform/ballerina-openapi/issues/614 to throw an error when this kind of scenarios found.

Reply:
Thanks @aneeshafedo 
I will try to find an official distribution on this. Or let me correct these for now.